docs/index.rst

```diff
@@ -7,15 +7,8 @@ API Reference
 .. toctree::
     :maxdepth: 2
 
-    gapic/v1beta1/api
-    gapic/v1beta1/reader
-    gapic/v1beta1/types
-
-    gapic/v1beta2/api
-    gapic/v1beta2/types
-
-    gapic/v1/api
-    gapic/v1/types
+    storage_v1/services
```

Comment:
We need to make sure the docs for the manual client(s) are included in here somewhere.

Reply:
Ah, I knew there was something left, should have written it down, thanks!